impl.go

```diff
@@ -0,0 +1,47 @@
+package localcache
+
+import (
+	"time"
+)
+
+type entry struct {
+	val    interface{}
+	expiry time.Time
+}
+
+type cache struct {
+	es map[string]*entry
+}
+
+func (c *cache) Get(key string) (interface{}, error) {
+	e, ok := c.es[key]
```

Comment:
If concurrent access this struct, what will happen? Could you please prevent this case?

Reply:
If simultaneously write value to same key, it possible to get different value from same key.
I will add mutex to lock write and read action.